source/crud/crud.rst

```diff
@@ -754,30 +754,38 @@ older sharded clusters.
 estimatedDocumentCount
 ~~~~~~~~~~~~~~~~~~~~~~
 
-On server versions greater than or equal to 4.9.0 (wire version 12 or higher),
-the estimatedDocumentCount function is implemented using the ``$collStats``
-aggregate pipeline stage with ``$group`` to gather results from multiple shards.
-As documented above, the only supported option is maxTimeMS::
-
-  pipeline = [
-    { '$collStats': { 'count': {} } },
-    { '$group': { '_id': 1, 'n': { '$sum': '$count' } } }
-  ]
-
-Similar to the count command, the estimated count of documents is returned
-in the ``n`` field. Implementations can assume that the document containing
-the single result of the aggregation pipeline is contained in the first batch of
-the server's reply to the aggregate command. It is not necessary to execute a getMore
-operation to ensure that the result is available.
-
-In the event this aggregation is run against a non-existent namespace, a NamespaceNotFound(26)
-error will be returned during execution. Drivers MUST interpret the server error code 26 as
-a ``0`` count.
-
-For server versions less than 4.9.0 (wire version 11 or under), the estimatedDocumentCount
-function is implemented using the ``count`` command with no query filter, skip,
-limit, or other options that would alter the results. Once again, the only supported
-option is maxTimeMS.
+The estimatedDocumentCount function is implemented using the ``count`` command
+with no query filter, skip, limit, or other options that would alter the
+results. The only supported options are listed in the
+``EstimatedDocumentCountOptions`` type defined above.
+
+Drivers MUST document that, due to a bug in versions 5.0.0-5.0.7 of MongoDB,
+estimatedDocumentCount was not included in v1 of the Stable API, so users of the
+Stable API are recommended to upgrade their server version to 5.0.8+ or set
+``apiStrict: false`` to avoid encountering errors when using
+estimatedDocumentCount.
+
```

Comment:
Can we just delete this entire paragraph? I think the final paragraph in this section better describes the server bug/regression and also clearly instructs drivers on how they should document this.

```suggestion
```

This may also address @JamesKovacs' concerns with referring to a "bug" here in the [preceding thread](https://github.com/mongodb/specifications/pull/1187/files#r854624143).

Reply:
Points noted. I missed that the later paragraph is mostly referring to "release notes". In that case I'll retract this suggestion.